features/step_definitions/user_steps.rb

```diff
@@ -139,8 +139,8 @@
 end
 
 Given /^I am logged out$/ do
-  visit destroy_user_session_path
-  visit destroy_admin_session_path
+  http_delete destroy_user_session_path
```

Comment:
Per @tickinginstant's comment elsewhere:

> I think it might be better to just replace the "I am logged out" step with something that resets cookies. Any of the `has_selector?` methods are going to cause delays one way or another.

We can have "I am logged out" use `page.driver.clear_cookies`.